WordPress/Classes/Services/MediaService.swift

```diff
@@ -2,7 +2,14 @@ import Foundation
 
 extension MediaService {
 
-    func refreshMediaStatus(onCompletion: (() -> Void)?, onError: ((Error) -> Void)?) {
+    /// /// This method check the status of all media objects and updates them to the correct status if needed.
```

Comment:
Nitpick – extra `///` here, and I'd change `check` to `checks` (that also applies to the comment in `MediaCoordinator` too) :)